gcloud/pubsub/topic.py

```diff
@@ -303,6 +303,31 @@ def set_iam_policy(self, policy, client=None):
             method='POST', path=path, data=resource)
         return Policy.from_api_repr(resp)
 
+    def test_iam_permissions(self, permissions, client=None):
```

Comment:
That would be a reasonable choice:  I'm on the fence about whether the improvement would be worth drifting from the method name used in the API.